comment/views/comments.py

```diff
@@ -59,14 +60,18 @@ def form_valid(self, form):
             parent=self.parent_comment,
             email=email,
             posted=time_posted
-            )
+        )
+
+        self.email_service = DABEmailService(_comment, self.request)
 
         if settings.COMMENT_ALLOW_ANONYMOUS and not user:
             # send response, please verify your email to post this comment.
-            response_msg = process_anonymous_commenting(self.request, _comment)
-            messages.info(self.request, response_msg)
+            self.email_service.send_confirmation_request()
+            messages.info(self.request, EmailInfo.CONFIRMATION_SENT)
         else:
             _comment.save()
+            if settings.COMMENT_ALLOW_SUBSCRIPTION:
+                self.email_service.send_notification_to_followers()
```

Comment:
same thing as the comment on the `api` `views` file.

Reply:
Note that the email service is not used for notifications only but also used for send confirmation_requests so I don't want to initialize the same service in 2 places.

Reply:
Not the best of decisions in my view.